agent/php_newrelic.h

```diff
@@ -181,7 +181,7 @@ typedef void (*nrphpfn_t)(INTERNAL_FUNCTION_PARAMETERS);
 typedef void(ZEND_FASTCALL* nrphpfn_t)(INTERNAL_FUNCTION_PARAMETERS);
 #endif /* PHP < 7.3 */
 
-#if ZEND_MODULE_API_NO <= ZEND_8_0_X_API_NO
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO
```

Comment:
Great catch!  The comment at the end of the block also needs to be updated for both of the cases: `#endif /* PHP <= 8.0 */` to `#endif /* PHP >= 8.0 */`

Reply:
Fixed!